modules/nf-core/gfastats/meta.yml

```diff
@@ -0,0 +1,71 @@
+name: "gfastats"
+description: |
+  A single fast and exhaustive tool for summary statistics and simultaneous *fa*
+  (fasta, fastq, gfa [.gz]) genome assembly file manipulation.
+keywords:
+  - genome assembly
+  - genome summary
+  - genome manipulation
+  - genome statistics
+tools:
+  - "gfastats":
+      description: "The swiss army knife for genome assembly."
+      homepage: "https://github.com/vgl-hub/gfastats"
+      documentation: "https://github.com/vgl-hub/gfastats/tree/main/instructions"
+      tool_dev_url: "https://github.com/vgl-hub/gfastats"
+      doi: "https://doi.org/10.1093/bioinformatics/btac460"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - assembly:
+      type: file
+      description: Draft assembly file
+      pattern: "*.{fasta,fastq,gfa}(.gz)?"
+  - out_fmt:
+      type: string
+      description: Output format (fasta, fastq, gfa)
+  - genome_size:
+      type: number
+      description: estimated genome size for NG* statistics (optional).
```

Comment:
```suggestion
      description: estimated genome size (bp) for NG* statistics (optional).
```

Reply:
Hehe. Good call. Ironically I made the same comment to the GFA author for their repo.